src/io/flutter/run/FlutterDebugProcess.java

```diff
@@ -15,12 +15,12 @@
 import com.intellij.ui.GuiUtils;
 import com.intellij.ui.content.Content;
 import com.intellij.xdebugger.XDebugSession;
-import com.jetbrains.lang.dart.ide.runner.server.vmService.DartVmServiceDebugProcessZ;
 import com.jetbrains.lang.dart.util.DartUrlResolver;
 import io.flutter.actions.ReloadFlutterApp;
 import io.flutter.actions.RestartFlutterApp;
 import io.flutter.run.daemon.FlutterApp;
 import io.flutter.run.daemon.RunMode;
+import io.flutter.server.vmService.DartVmServiceDebugProcessZ;
```

Comment:
If we are not going to merge into the Dart plugin then we could drop the Z on this class name. I think I added it to distinguish it from the superclass in a way that would invite merging (but I'm not certain about that).